05-cond.md

```diff
@@ -199,12 +199,18 @@ and false statements. However, they aren't the only values in Python that are tr
 > (Note that if the body of a conditional is a single statement, we can write it on the same line as the `if`.)
```

Comment:
None of the statements below do this any more (although I wouldn't encourage it anyway.)